libs/lib_parsing/Pos.ml

```diff
@@ -128,7 +128,7 @@ let converters_of_arrays ?(file = "<unknown>") line_arr col_arr :
   (* len1 and len2 should be equal but we're playing it safe *)
   let len = min len1 len2 in
   match len with
-  | 0 ->
+  | 2 ->
```

Comment:
What is this change?

Reply:
add a comment about it then, and the result of your investigation, and a big TODO about it